fbw-a32nx/src/systems/instruments/src/EFB/Store/features/simBrief.ts

```diff
@@ -119,6 +119,8 @@ export const initialState: {data: SimbriefData} = {
         loadsheet: '',
         costInd: '',
     },
+    payloadImported: false,
+    fuelImported: false,
```

Comment:
In the longer term, we may have to consider a bit more whether managing the state like this is a good approach, or if it should be a simvar or some other form of persistence as if I'm not wrong, currently this would maintain the same behaviour that when the EFB is powered off, that all simbrief import is lost. 

Maybe not in this PR though.

Reply:
Yes, these two new States would the same Behavior! (One Reason I've added them there. The other being: I just don't know how/where else^^)
So it fits in the current State the User already knows. Power Off => SimBrief Data lost. Power On => It is imported again. I just "hooked on" to that. (The Fuel & Payload is also then reimported)

To be clear: I'm totally clueless. So I would say: definitely not in this PR respectively not by me 😅